docs/index.md

```diff
@@ -1,6 +1,18 @@
 # Welcome to Subatomic!
 
-// TODO
+Subatomic is toolset designed to automate the provisioning and use of your developement environment from infrastructure through to CI/CD pipelines and associated monitoring. The goals of the project are to remove the hassle and cost of repeating the processes required to get new projects up an running from scratch to being to deployed into multiple environments. The project home can be found [here](https://github.com/absa-subatomic).
```

Comment:
Subatomic is **a** toolset designed to automate the provisioning and use of your **development** environment from infrastructure through to CI/CD pipelines and associated monitoring. The goals of the project are to remove the hassle and cost of repeating the processes required to get new projects up **and** running from scratch to being to**remove** deployed into multiple environments. The project home can be found [here](https://github.com/absa-subatomic).